### DISCUSSION TITLE
EQ1 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#68] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/EQ1.html
Author: @kathypinzon 
Primary Reviewer: @tdegeorge

**Comment by tdegeorge** [comment]
For Activity 1.1.2: I think part c is missing the MC options? Also wondering if we shoud scaffold the fraction ones too. For instance, have part c be something that has only one fraction and then move part c to part d (that has a combination of fractions in one problem).

**Reply by kathypinzon** [reply]
made a new activity just for the fractions and tried to scaffold a bit more.

**Comment by tdegeorge** [comment]
For Activity 1.1.3: Maybe in the intro, we can say something to the effect that linear equations don't always have just one answer (or maybe a remark beforehand?)

Maybe switch b and c so that no solution comes first.

Part d technically has 2 questions. Maybe split them into two different tasks? OR maybe combine the two questions into one? Like "what happens to the x variable when the solution to the linear equations has either no solution or infinitely many solutions?"

**Comment by tdegeorge** [comment]
Definition 1.1.4: There is an extra period after "identity equation."

**Comment by tdegeorge** [comment]
For Activity 1.1.5: Is there a way to put the graphs in as multiple-choice answers (rather than have them draw it themselves on the first time they have to do it)? It might also be better to have them graph it first and then write the answer in interval notation (switch c and d).

Do we need a remark or something to remind students about interval notation?

**Reply by kathypinzon** [reply]
working on the graph part....
interval notation is in the Readiness materials so I don't think we need to address it here.

**Comment by tdegeorge** [comment]
For Activity 1.1.6: Do we also want to include graphs for this activity as well? Only if that is easy to do....:)

**Comment by tdegeorge** [comment]
Activity 1.1.10: I think this activity could be scaffolded a little bit more. For part b, we could have them solve the two separate inequalities and then part c would be to graph the solutions on two separate number lines. Then, in part d, they could be asked how the solutions overlap and how to express both solutions on one number line (leading them to the interval notation as well). 

Could we maybe add something at the end of the activity about how they could solve the compound inequality as a single linear inequality? Like have them attempt it and then end the section with the remark at the end.

**Reply by kathypinzon** [reply]
updated - just have to finish the number line part.
